cosmos/cosmos-sdk
# Fix multisig LegacyAminoPubKey Amino marshaling

## Description



revert: #8794 (that was a temporary workaround)
closes: #8776

PR description in `crypto/keys/multisig/amino.go` comments

To test, I did:
- take the JSON file by Akhil in #8776 
- temporarily change the default bech32 prefix from "cosmos" to "akash" to match the JSON file
- run `simd migrate v0.40 <json_file>`
- make sure the `account_number:"90"` has correct NON-EMPTY multisig field.

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.
